foundryup/foundryup

```diff
@@ -32,6 +32,10 @@ main() {
       -j|--jobs)        shift; FOUNDRYUP_JOBS=$1;;
       --arch)           shift; FOUNDRYUP_ARCH=$1;;
       --platform)       shift; FOUNDRYUP_PLATFORM=$1;;
+      --self-update)
+        self_update
+        exit 0
+        ;;
```

Comment:
I think we can simply use `update` here, mirroring `rustup`, please use the formatting and handling of the other cases